lib/datasource/metadata.ts

```diff
@@ -246,12 +246,18 @@ export function addMetaData(
   }
 
   // Clean up any empty urls
-  const urls = ['homepage', 'sourceUrl', 'changelogUrl', 'dependencyUrl'];
-  for (const url of urls) {
-    if (is.string(dep[url]) && validateUrl(dep[url].trim())) {
-      dep[url] = dep[url].trim();
+  const urlKeys: (keyof ReleaseResult)[] = [
```

Comment:
Remove?

Reply:
Yup

Reply:
Not sure how to make it work without this type declaration, though